core/src/main/gamelogic/Goal.kt

```diff
@@ -0,0 +1,44 @@
+package gamelogic
+
+import gamelogic.map.Continent
+import java.lang.Exception
+
+abstract class SubGoal {
+    abstract fun achieved(player:PlayerInfo, referee:Referee):Boolean
+}
+
+class Goal(val subGoals: List<SubGoal>) : SubGoal() {
```

Comment:
las subGoals pueden ser una `Collection` en vez de una `List`? O necesitás los índices de los 
subGoals? Además, tienen que ser public?
```suggestion
class Goal(private val subGoals: Collection<SubGoal>) : SubGoal() {
```

Reply:
Lo mismo con todos los otros parámetros del archivo, no pueden ser `private`?

Reply:
Un año de python me mal acostrumbró. Es peor de lo que parece.